src/components/application_manager/src/message_helper/message_helper.cc

```diff
@@ -739,6 +739,26 @@ const VehicleData& MessageHelper::vehicle_data() {
   return vehicle_data_;
 }
 
+std::string MessageHelper::AppHMITypeToString(
+    mobile_apis::AppHMIType::eType type) {
+  using namespace ns_smart_device_link::ns_smart_objects;
+  const char* str = 0;
+  if (EnumConversionHelper<mobile_apis::AppHMIType::eType>::EnumToCString(
+          type, &str)) {
+    return str;
+  }
+  return std::string();
+}
+
+mobile_apis::AppHMIType::eType MessageHelper::StringToAppHMIType(
```

Comment:
```
template<typename T>
T StringToEnum(std::string )

template<typename T>
string EnumTOString( type )
```
I would even make it  not a static of message helper. Just funciotns in message_helper header

Reply:
Such approach allows us not to mock this functions. I do not see the reason to mock it. because it should use generated code from API

Reply:
Done in commit 274f459241618e4418b4d787c40f8ee27c4f81a1